app/src/main/java/de/westnordost/streetcomplete/osm/cycleway/CyclewayCreator.kt

```diff
@@ -185,6 +196,11 @@ private fun CyclewayAndDirection.applyTo(tags: Tags, isRight: Boolean, isLeftHan
     if (!touchedSegregatedValue) {
         tags.remove("$cyclewayKey:segregated")
     }
+    // no matter what option was chosen, if it's not SIDEWAY_OK, remove that cycling is signed and ok on sidewalk if it's there
+    if (cycleway != SIDEWALK_OK && tags.containsKey("sidewalk:$side:bicycle") && tags["sidewalk:$side:bicycle"] == "yes" && tags.containsKey("sidewalk:$side:bicycle:signed")) {
```

Comment:
```suggestion
    if (cycleway != SIDEWALK_OK && tags["sidewalk:$side:bicycle"] == "yes" && tags["sidewalk:$side:bicycle:signed"] == "yes") {
```

(Or was there a reason why you are not checking the value of `sidewalk:$side:bicycle:signed`?)